### ISSUE TITLE
CC messages are not forwarded

### ISSUE THREAD

**Issue by jofemodo** [BramGiesen/arpeggiator_LV2#11]
Hi @BramGiesen !

We have recently integrated the arpeggiator in the zynthian platform and it works quite fine except CC messages (and probably others) are not forwarded. This makes the MIDI routing more complex ...

**Comment by BramGiesen** [comment]
Hi @jofemodo,

At the moment the plugin only uses the note on and off messages when it's turned on. It would indeed be good to forward all other message's to the output directly. I have made start for this, but it will need a few extra changes. I will have a look at this soon.

**Comment by BramGiesen** [comment]
I've created a new version of this [arpeggiator](https://github.com/moddevices/mod-arpeggiator-lv2) using the DPF framework. This version does include a MIDI pass through for all messages that are not used by the arpeggiator.

**Comment by jofemodo** [comment]
Great! I will test it ASAP ...
Must i understand that development has moved to the mod repo?

**Comment by BramGiesen** [comment]
Yes, for now it's moved over there. I might do some development on this repo in the future if I want to do some things that would be different from the MOD version. But for now I don't have any plans for this.
